Cargo.toml

```diff
@@ -65,7 +62,7 @@ sqlite = ["rusqlite", "ahash"]
 compact_filters = ["rocksdb", "socks", "lazy_static", "cc"]
 key-value-db = ["sled"]
 all-keys = ["keys-bip39"]
-keys-bip39 = ["tiny-bip39", "backtrace"]
+keys-bip39 = ["bip39"]
```

Comment:
Question: Instead of this chained features can we make it `all-keys=["bip39"]`?

Reply:
I believe the reason `keys-bip39` is there is in case we switched from `tiny-bip39` to some other bip39 crate (as you're doing now) then end users wouldn't need to change the feature name in their projects. Hopeful we won't need to change bip39 crates again, but just in case, I think it's OK to leave the `keys-bip39` feature here as it is.